schemas/context/segmentmembershippropensitypayload.example.1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "xdm:payloadValue": 0.5,
+  "xdm:payloadType": "https://ns.adobe.com/xdm/context/segmentMembershipPropensityPayload"
```

Comment:
We need someway to embed the payload type with the payload if we want to attempt to validate the structure is correct.